STORM-IRIT/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM ubuntu:18.04
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update
+RUN apt-get install -y cmake pkg-config  wget unzip
+
+#CGAL installation:
+RUN apt-get install -y libgmp-dev libmpfr-dev libboost-all-dev
+
+WORKDIR /root
+
+RUN wget https://github.com/CGAL/cgal/releases/download/releases%2FCGAL-5.0.2/CGAL-5.0.2.zip
+
+RUN unzip CGAL-5.0.2.zip
```

Comment:
Remember that each command run - is a layer. It means that is better to install cgal in one command (via &&)